device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O32/pg_profile_lookup.ini

```diff
@@ -0,0 +1 @@
+../Mellanox-SN4700-C128/pg_profile_lookup.ini
```

Comment:
Link to this file https://github.com/sonic-net/sonic-buildimage/blob/master/device/mellanox/x86_64-nvidia_sn4280-r0/ACS-SN4280/pg_profile_lookup.ini